x-pack/plugins/observability_solution/synthetics/server/saved_objects/saved_objects.ts

```diff
@@ -37,6 +40,7 @@ export const registerSyntheticsSavedObjects = (
   savedObjectsService: SavedObjectsServiceSetup,
   encryptedSavedObjects: EncryptedSavedObjectsPluginSetup
 ) => {
+  savedObjectsService.registerType(LEGACY_PRIVATE_LOCATIONS_SAVED_OBJECT_TYPE);
   savedObjectsService.registerType(PRIVATE_LOCATIONS_SAVED_OBJECT_TYPE);
```

Comment:
```suggestion
  savedObjectsService.registerType(PRIVATE_LOCATION_SAVED_OBJECT_TYPE);
```
I'd like to see this variable singular as well.